src/test/data/ManualTesting/TestScript.md

```diff
@@ -0,0 +1,199 @@
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | To undo last action | undo | the last edit command should be undone
+2 | To redo last action | redo | the last edit command should be redone again
+
+TestCase ID : TC010 <br>
+Title : Changing/Check save data location <br>
+Description : Tester should be able to change the save data location <br>
+Format : <br>
+`setdatadir C:\Users` will change the directory on a windows platform
+`setdatadir ./path/to/new/location/on/unix/platform/` will change the location on a unix platform <br>
+Description : Tester should be able change the location of the save data location. <br>
+Assumption : Tester should be able to provide a valid file path, not doing so will result in a invalid file path message
+Tester should provide a file path that do not require system permission (e.g saving inside a C drive C:\ )
+
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | change the test data to a new location | refer to the setdatadir command format explained in this test case | The system should display the location of the new sav data. the new data should be created physically on the new location as well
+
+
+TestCase ID :TC011 <br>
+Title : Clear the entire saved data <br>
+Description : Tester should be able to clear the entire saved data completely. This action can be undo. <br>
+
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | clear everything in TaskTracker | clear | the list should show nothing after this command
+2 | undo the clear command | undo | the list should show everything back before clearing
+
+TestCase ID : TC012 <br>
+Title : Auto hide the program and wake up the program from background. <br>
+Description : Tester should hide the program into background and wake it up when needed.
+Hot Key: <br>
+`CTRL + SPACE`<br>
+This hot key will only work on windows system.
+
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | hide the app in background | CTRL + SPACE | the TaskTracker will be hiden into backgound
+2 | wake up the app from background | CTRL + SPACE | the TaskTracker window will be shown again
+
+TestCase ID : TC013 <br>
+Title : Auto complete. <br>
+Description: Tester should use `TAB` to do auto-completing when they are entering command keyword and edit certain fileds of some tasks.
+Hot Key: <br>
+`TAB`<br>
+
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | Go to the Command box on the top of the app | | User should be able to type anything here
+2 | find task whose name contains `Unicycling` keyword | find 'Unicycling' | TaskTracker will list all tasks whose name contains the keyword
+3 | show the command list in command box | TAB | the command list will shown
```

Comment:
With the command box empty but selected, press TAB.

A list of commands will pop up.